modules/runner/src/main/java/org/apache/ignite/internal/app/IgnitionImpl.java

```diff
@@ -108,10 +115,21 @@ else if (jsonStrBootstrapCfg != null)
 
         var serializationRegistry = new MessageSerializationRegistry();
 
+        String localNodeName = netConfigurationView.name();
+
+        if (StringUtil.isNullOrEmpty(localNodeName)) {
+            localNodeName = "Node: " + netConfigurationView.port();
```

Comment:
Should we use default value provided by configuration schema instead.

Reply:
localNodeName this is a default value, which was provided into bootstrap.